docs/user/templates/11_model_observability.md

```diff
@@ -37,13 +39,17 @@ Beside changes in signature, you can see some of those methods returning new typ
 | `session_id` | str | Identifier for the request. This value will be used together with `prediction_ids` as prediction identifier in model observability system  |
 
 `ModelInput` data is essential for model observability since it contains features values and identifier of prediction. Features values are used to calculate feature drift, and identifier is used as join key between features, prediction data with ground truth data. On the other hand, `ModelOutput` is the class that represent raw model prediction output, not the final output of PyFunc model. `ModelOutput` class contains following fields:
+
 | Field | Type | Description |
 |-------|------|-------------|
 | `prediction` | Values | `predictions` contains prediction output from ml_predict, it may contains multiple columns e.g for multiclass classification or for binary classification that contains prediction score and label |
 | `prediction_ids` | List[str] | Unique identifier for each row in prediction output |
 
 Same like `ModelInput`, `ModelOutput` is also essential for model observability, it can be used to calculate prediction drift but more importantly it can calculate performance metrics.
 
+### Standard Model
+Only xgboost model is supported at the moment. To enable model observability, user will need to enable request response logging for the model. 1% of the predictions will be sampled, and this is not configurable at the moment.
```

Comment:
can u add detail on what is changed on the caller side when enable model observability for standard model e.g payload that contains session id and row id?